katana_400_6m90G:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50 

  # Position Controllers ---------------------------------------
  katana_motor1_pan_joint_position_controller:
    type: position_controllers/JointTrajectoryController
    joint: katana_motor1_pan_joint
    pid: {p: 6.0, d: 4.0, i: 0.0, i_clamp: 0.0}

  katana_motor2_lift_joint_position_controller:
    type: position_controllers/JointTrajectoryController
    joint: katana_motor2_lift_joint
    pid: {p: 4.0, d: 3.0, i: 0.0, i_clamp: 0.0}

  katana_motor3_lift_joint_position_controller:
    type: position_controllers/JointTrajectoryController
    joint: katana_motor3_lift_joint
    pid: {p: 2.0, d: 1.5, i: 0.0, i_clamp: 0.0}

  katana_motor4_lift_joint_position_controller:
    type: position_controllers/JointTrajectoryController
    joint: katana_motor4_lift_joint
    pid: {p: 1.0, d: 1.0, i: 0.0, i_clamp: 0.0}

  katana_motor5_wrist_roll_joint_position_controller:
    type: position_controllers/JointTrajectoryController
    joint: katana_motor5_wrist_roll_joint
    pid: {p: 0.1, d: 0.05, i: 0.0, i_clamp: 0.0}


  


